extension/src/popup/views/SignTransaction.tsx

```diff
@@ -95,7 +123,9 @@ export const SignTransaction = () => {
   );
 
   const punycodedDomain = getPunycodedDomain(domain);
-  const { _fee, _operations } = transaction;
+  const { _fee, _operations, _memo, _sequence } = transaction;
```

Comment:
ah, so I thought I remembered solving the issue with Memo text coming in as a buffer and I tracked down the PR where we removed it: https://github.com/stellar/lyra/pull/24

Looks like we had a convo about it but i cannot for the life of me recall what this was about :D
Do you remember?

Reply:
I'm not 100% sure but it looks like the very first [design](https://projects.invisionapp.com/prototype/Lyra-Lite-ck60vjr6j003ky301z14bfmta/play/781cb375) had the memo but the revised design after that didn't. Then orbitlens suggested including MEMO on 08/19.

Reply:
Added MEMO! Thanks @piyalbasu